grakn-graql-shell/src/main/java/ai/grakn/graql/JsonSession.java

```diff
@@ -145,6 +145,7 @@ void sendJson(Json json) throws WebSocketException, IOException {
     }
 
     @OnWebSocketClose
+    @SuppressWarnings("unused") //Used through socket connection.
```

Comment:
IntelliJ magic: you can actually tell IntelliJ that `@OnWebsocketClose` marks an entry point if you want instead.